shared/ui-components/src/helpers/formOptions.tsx

```diff
@@ -5,6 +5,12 @@ interface FormOptionsProps {
   options: string[]
   keyPrefix: string
 }
+
+interface CheckboxGroupItem {
+  name: string
+  checked?: boolean
+}
+
 export const contactPreferencesKeys = ["email", "phone", "letter", "text"]
 
 export const relotionshipKeys = [
```

Comment:
I know this isn't your change, but while you're in here can you fix this typo? (relotionship)